agent-install/README.md

```diff
@@ -30,30 +30,32 @@ This script:
 
 ## Requirements
 
-Currently supported operating systems and architectures:
+Operating systems and architectures explicitly supported by the installation script.  Please note that an environment must be both supported by the installation script, and [an installation package must be available](https://github.com/open-horizon/anax/releases), in order to successfully install the agent.  Environments that meet both criteria are noted in **bold** below.
 
 - Device
-  - Ubuntu bionic, xenial, focal, jammy, noble
-    - amd64, arm64
-  - Raspbian stretch, buster, bullseye, bookworm
-    - armhf, arm64
-  - Debian stretch, buster, bullseye, bookworm
-    - amd64, armhf, arm64, s390x
-  - RHEL 8.1 - 8.8, 9.0 - 9.5
-    - amd64, ppc64le, aarch64, riscv64, s390x
-  - CentOS 8.1 - 8.5 (via Docker)
-    - amd64, ppc64le, aarch64, riscv64
-  - Fedora 36 - 41
-    - amd64, ppc64le, aarch64, riscv64
+  - Ubuntu: xenial (16.x), bionic (18.x), focal (20.x), jammy (22.x), noble (24.x)
+    - **amd64, arm64, s390x**
+  - Raspbian/RaspberryPi OS: stretch (9), buster (10), bullseye (11), bookworm (12)
+    - **armhf, arm64**
+  - Debian: stretch (9), buster (10), bullseye (11), bookworm (12)
+    - **amd64, armhf, arm64, s390x**
+  - RHEL: 7.6, 7.9, 8.1 - 8.5 (via Docker), 8.6 - 8.10 and 9.0 - 9.5 (via Podman 4.x)
+    - **amd64, ppc64le**, aarch64, riscv64, **s390x**
+  - CentOS: 8.1 - 8.5 (via Docker)
+    - **amd64, ppc64le**, aarch64, riscv64
+  - Fedora: 32, 35 - 43
+    - **amd64, ppc64le**, aarch64, riscv64
   - macOS
-    - amd64, M1, M2
+    - **amd64, M1, M2, M4**
 - Cluster
-  - OCP 4.5, 4.6, 4.7, 4.8
-    - amd64, ppc64le
+  - OCP: 4.5, 4.6, 4.7, 4.8
```

Comment:
I think it would be better to remove the versions from OCP... All of these listed are out of support... Additionally, we don't list versions for k3s and microk8s.

Reply:
We may want to spell out OCP as OpenShift Container Platform too

Reply:
@dlarson04 I've spelled out OCP, removed OCP versions, and added a statement that we intended to work with supported versions of the specified kube distributions on the supported architectures.  Does this resolve the points you mentioned?